### PR TITLE
bump version to 2.9.0 & update release notes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,8 +3,60 @@ For more detailed information, please see the git log.
...
+    BayPass (#3451), Bazel (#3379), Blender (#3553, #3558), bwakit (#3567), BXH_XCEDE_TOOLS (#3410), CastXML (#3403),
+    CHASE (#3304), configparser (#3368, #3424), configurable-http-proxy (#3380), cppcheck (#3508), CRPropa (#779),
+    DicomBrowser (#3432), DMTCP (#3422), entrypoints (#3368, #3424), f90cache (#3570), fastPHASE (#3343),
+    fastQValidator (#3192), FFindex (#1135), FragGeneScan (#1198), gdc-client (#3399), gflags (#3417), glog (#3417),
+    GRIT (#3561), H5hut (#3431), HAPGEN2 (#3344), HEALPix (#779), IMOD (#1187, #3347), IronPython (#607),
+    jhbuild (#3476), jupyterhub (#3380), Keras (#3581), khmer (#1158), LeadIT (#3345, #3599), LevelDB (#3417),
+    libbitmask (#3481), libcpuset (#3481), LMDB (#3417), log4cplus (#1136), MACH (#3346), Mako (#3460), Maq (#3428),
+    MetaGeneAnnotator (#3307), Metal (#3324), Mono (#607), MyMediaLite (#607), nco (#2575), nose-parameterized (#3579),
+    OpenEXR (#3553), OpenImageIO (#3553), path.py (#3368, #3424), PCRE2 (#3325), pftoolsV3 (#3317), PHASE (#3385),
+    PLAST (#3288), PLINKSEQ (#3402), POV-Ray (#3551), ProbABEL (#3108), prompt-toolkit (#3368, #3424),
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: nice, glad to see the POV-Ray being included here; it likely addresses that historic PR!

Comment (fgeorgatos) [inline RELEASE_NOTES]: and also other nice ones here, such as ironpython, mono, keras, lmdb... there's something for everybody, good!

File: RELEASE_NOTES [hunk 2]
```diff
@@ -3,8 +3,60 @@ For more detailed information, please see the git log.
...
+    OpenMPI 2.0.1, ParaView 5.1.2, PGI 16.7, QuantumESPRESSO 5.4.0, Qt5 5.7.0, R-bundle-Bioconductor 3.3, VTK 7.0.0,
+    Yade 2016.06a
+- various enhancements, including:
+    - adjust PSI4 easyconfigs for updated easyblock (#3312)
+    - clean up libxml2 easyconfigs according to updated libxml2 easyblock (#3479, #3509)
+    - significantly speed up verifying of dumped easyconfig by resorting to 'shallow' parsing (#3520)
+    - include sanity checks for all MATIO config files (#3528)
+    - remove --with-tcl-config/--with-tk-config from R easyconfig, already done in R easyblock (#3580)
+- various bug fixes, including:
+    - disable testing in all ParaView 4.4.0 easyconfigs, required download is too much of a PITA (#3178)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: 1 idea, for the future intern:
* make all these difficult tests being able to invoke with some special parameter, fi. --pedantic-testing

File: setup.py [hunk 3]
```diff
@@ -47,7 +47,7 @@
 # recent setuptools versions will *TRANSFORM* something like 'X.Y.Zdev' into 'X.Y.Z.dev0', with a warning like
 #   UserWarning: Normalizing '2.4.0dev' to '2.4.0.dev0'
 # This causes problems further up the dependency chain...
-VERSION = '2.9.0.dev0'
+VERSION = '2.9.0'
```
Comment (fgeorgatos) [inline setup.py]: ie. no comments, thanks for the release!

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos!

**Pull request merged**